core/src/main/java/apoc/meta/Meta.java

```diff
@@ -750,7 +750,7 @@ private void setIncomingRelationships(Map<String, Object> nodes, Map<String, Lis
         for(String entityName : metaData.keySet()) {
             Map<String, MetaResult> entityData = metaData.get(entityName);
             Map<String, Object> entityProperties = new LinkedHashMap<>();
-            if (entityData.isEmpty()) {
+            if (entityData.isEmpty() && metaStats.labels.containsKey(entityName)) {
```

Comment:
non ho capito perché?

Reply:
Perché con un basso sample, può capitare che alcune relazioni non vengano "samplate", e quindi non venga popolato `entityData` ma che in `metaStats` sia presente l'entità. Perciò devo verificare che non sia una label.
Ho fatto `metaStats.labels` e non `metaStats.relTypes`, perché il primo è `["LabelA", "LabelB",...]` mentre il secondo è tipo `["(:LabelA)-[:REL_0]->()", "()-[:REL_0]->(:LabelA)", "()-[:REL_0]->()"...]`, quindi il primo è più semplice da verificare